src/org/usfirst/frc/team4915/steamworks/commands/DriveStraightCommand.java

```diff
@@ -31,7 +31,7 @@ public DriveStraightCommand(Drivetrain drivetrain, double inches)
         m_pidController = new PIDController(k_P, k_D, k_I, k_F, this, this);
         m_pidController.setOutputRange(-1, 1); // Set the output range so that this works with our PercentVbus turning method
         m_pidController.setInputRange(-5 * Math.abs(m_revs), 5 * Math.abs(m_revs)); // We limit our input range to revolutions, either direction
-        m_pidController.setPercentTolerance(1); // This is the tolerance for error for reaching our target
+        m_pidController.setAbsoluteTolerance(0.053); // This is the tolerance for error for reaching our target, targeting one inch
```

Comment:
`1 / (3 * (2 * Math.PI))` is a little longer but slightly easier to read 'one inch over the circumference of a 3 inch radius wheel'